src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
+  background: $card-background-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &:nth-child(1n) {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.